drivers/net/Kconfig

```diff
@@ -35,6 +35,8 @@
 #
 menuconfig SLIP
 	bool "SLIP driver"
+	default y if NET_QEMU_SLIP
```

Comment:
SLIP has a prompt.

This means it is user-settable and the value can be configured to be different from the default.

Is it valid to have NET_QEMU_SLIP, but not SLIP?

If not you need to either get rid of the prompt, or use select, or use depends.

Reply:
The NET_QEMU_SLIP requires SLIP. It is possible to use SLIP but not use the networking part like we do now.